### DISCUSSION TITLE
Attribute based selling

### DISCUSSION THREAD

**Discussion by luisweir** [OAI/sig-travel#15] [General]
New and exciting trend in hospitality (already established on Air)

**Comment by luisweir** [comment]
An article which I found extremely useful to understand the concept.

https://www.hospitalityupgrade.com/techTalk/February-2021/Definitely-Doug-2-12-21/

**Reply by kinlane** [reply]
OpenTravel is working on as part of 2.0.

**Reply by kinlane** [reply]
Oaky - https://www.oaky.com/

**Reply by kinlane** [reply]
Ideas - https://ideas.com/

**Reply by kinlane** [reply]
Hotel Central Reservations System (CRS) - https://www.amadeus-hospitality.com/central-reservations-system-software/crs/

**Reply by kinlane** [reply]
Oracle Hotel Property Management and POS Solutions - https://www.amadeus-hospitality.com/central-reservations-system-software/crs/
